Core/RestaurantReservation.Application/Services/LoginService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using RestaurantReservation.Application.Contracts.IService;
+using RestaurantReservation.Application.Contracts.IServices;
+using RestaurantReservation.Application.DTOs;
+using RestaurantReservation.Domain.Entities;
+
+namespace RestaurantReservation.Application.Services
+{
+    public class LoginService : ILoginService
+    {
+        private readonly string _pepper;
+        private readonly int _iteration = 3;
+        private readonly IAuthenticationService _authenticationService;
+
+        private readonly List<User> _userRepository = new List<User>()
+        {
+            new User {Name = "Tasbeeh", HashedPassword = "KQC76m3QBD2+E/SZ5tWLzxAb2eEaNUjFRmHvg9kicNo=", PasswordSalt="112255" },
+            new User {Name = "Israa", HashedPassword = "MfzC1OLrzluRiDKKKK0sAM4E5NNgTSCAbiawZT1oojE=", PasswordSalt="5555" },
+        };
+
+        public LoginService(IAuthenticationService authenticationService)
+        {
+            _pepper = Environment.GetEnvironmentVariable("PasswordHashExamplePepper");
+            _authenticationService = authenticationService;
+        }
+        public string? Login(LoginUserDTO loginUser)
+        {
+            var user = _userRepository
+                .FirstOrDefault(u => u.Name == loginUser.Name);
+
+            if (user == null)
+                return null;
```

Comment:
Null indicates two different errors, let's throw an exception or handle it in a different way.